frontend_tests/node_tests/unread.js

```diff
@@ -191,15 +191,15 @@ var zero_counts = {
 
     var num_msgs = 10000;
     var i;
-    for (i = 0; i < num_msgs; ++i) {
+    for (i = 0; i < num_msgs;i+=1) {
         message.id = i+1;
         unread.process_loaded_messages([message]);
     }
 
     count = unread.num_unread_for_subject('social', 'lunch');
     assert.equal(count, num_msgs);
 
-    for (i = 0; i < num_msgs; ++i) {
+    for (i = 0; i < num_msgs;i+=1) {
```

Comment:
This whitespace for these for looks still looks wrong; there should be a space after the `;`.